package.json

```diff
@@ -37,8 +37,8 @@
   "scripts": {
     "predeploy": "npm run build",
     "deploy": "gh-pages -d build",
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "REACT_APP_GIT_SHA=$(git rev-parse --short HEAD) REACT_APP_BUILD_DATE=$(date --utc '+%Y%m%d') react-scripts start",
+    "build": "REACT_APP_GIT_SHA=$(git rev-parse --short HEAD) REACT_APP_BUILD_DATE=$(date --utc '+%Y%m%d') react-scripts build",
```

Comment:
Why not `'+%Y-%m-%d'`? Should make it way easier to read.

Reply:
Fair enough. Changed the format.